config/locales/en/providers.yml

```diff
@@ -820,21 +820,15 @@ en:
       show:
         error: Select yes if they have received other benefits
     income_summary:
-      income_type_item:
```

Comment:
Are these handled elsewhere now?  
I always get the fear when locale stuff is deleted, not moved! :)

Reply:
Sorry, was probably a bit heavy handed of me. I couldn't see where they were being used, but I can add them back in to be on the safe side

Reply:
We do run an i18n test for expected keys being missing and if that didn't trigger it's probably okay :+1: